ui/app/components/app/sidebars/sidebar.component.js

```diff
@@ -59,9 +59,9 @@ export default class Sidebar extends Component {
           transitionEnterTimeout={300}
           transitionLeaveTimeout={200}
         >
-          { sidebarOpen && !sidebarShouldClose ? this.renderSidebarContent() : null }
```

Comment:
Instead of removing the `&& !sidebarShouldClose` check here, let's update the conditions under which `sideShouldClose` is true.

Off the top of my head, I think the line in `routes.js` that reads
`sidebarShouldClose={sidebarTransaction && !submittedPendingTransactions.find(({ id }) => id === sidebarTransaction.id)}`
could be updated to
`sidebarShouldClose={sidebarTransaction && !sidebarTransaction.status === 'failed' && !submittedPendingTransactions.find(({ id }) => id === sidebarTransaction.id)}`